src/main/java/mil/dds/anet/resources/PositionResource.java

```diff
@@ -224,4 +224,65 @@ public Integer deletePosition(@GraphQLRootContext Map<String, Object> context,
     return dao.delete(positionUuid);
   }
 
+  @GraphQLMutation(name = "mergePosition")
+  public Position mergePosition(@GraphQLRootContext Map<String, Object> context,
+      @GraphQLArgument(name = "loserUuid") String loserUuid,
+      @GraphQLArgument(name = "winnerPosition") Position winnerPosition) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    final Position existingPos = dao.getByUuid(loserUuid);
+
+    // Check that given two position can mergeable
+    arePositionsMergeable(existingPos, winnerPosition);
+    assertCanUpdatePosition(user, winnerPosition);
+    validatePosition(user, winnerPosition);
+
+    // Delete loser position's code information
+    int rowNums = dao.cleanLoserPositionCode(loserUuid);
+    if (rowNums == 0) {
+      throw new WebApplicationException("Couldn't process merge operation", Status.NOT_FOUND);
```

Comment:
"Couldn't process merge operation, because ... "

For the user to understand the error message easily